App.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Innocent's Portfolio</title>
+        <link rel="stylesheet" href="App.css">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap" rel="stylesheet">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    </head>
+    <body class="container">
+        <header class="header">
+            <nav class="nav">
+                <h1 class="nav-title">Innocent N.</h1>
+                <a href="#">
+                    <img src="./images/menu.png" alt="menu-icon" class ="nav-menu"/>
+                </a>      
```

Comment:
- [ ] Replace this `a` tag with a `nav` because the hamburger button is a navigation item